pkgs/applications/misc/calibre/default.nix

```diff
@@ -51,6 +94,11 @@ stdenv.mkDerivation (finalAttrs: {
       url = "https://raw.githubusercontent.com/debian-calibre/calibre/debian/${finalAttrs.version}+ds-1/debian/patches/hardening/0007-Hardening-Qt-code.patch";
       hash = "sha256-a6yyG0RUsQJBBNxeJsTtQSBV2lxdzz1hnTob88O+SKg=";
     })
+    (fetchpatch {
+      name = "build-from-git-without-internet";
+      url = "https://github.com/kovidgoyal/calibre/commit/4b7c79f71821e9d13edff5d04249c5decf1093a0.patch";
+      hash = "sha256-1czbc5Z5BsvtyVWnQwFH3Wsn+31RfahjulwtXHJVQKA=";
+    })
```

Comment:
Is there any chance of upstreaming this to calibre in the future?

Reply:
I just made PR to calibre (https://github.com/kovidgoyal/calibre/pull/2193).